Makefile

```diff
@@ -122,7 +124,7 @@ generate: build-local embedmd
 	@$(GOPATH)/bin/embedmd -w `find . -path ./vendor -prune -o -name "*.md" -print`
 
 embedmd:
-	go get github.com/campoy/embedmd@v1.0.0
+	GO111MODULE=on go get github.com/campoy/embedmd@v1.0.0
```

Comment:
I am not sure if this is desirable. This changes the `go mod` artifacts and will affect the `go mod verify` checks in the CI.

Reply:
Yep you are right. But otherwise, go get fails.

Reply:
I am not sure we want to add this `@go mod tidy` to the `doccheck` target. Can we try this with `GO111MODULE=off` instead? I don't see why this needs to be added to the `go mod` artifacts, assuming we are using `embdedmd` as a a CLI convenience in CI.

Reply:
Interestingly enough with "off" hit this problem locally: `cannot use path@version syntax in GOPATH mode`

https://github.com/golang/go/issues/29415

Reply:
But I agree that was a hack, doing go mod tidy, I am sure there is a better way! :)